pimcore/models/Object/ClassDefinition/Data/Classificationstore.php

```diff
@@ -1004,12 +1004,17 @@ public function setLocalized($localized)
         $this->localized = $localized;
     }
 
-
+    /**
+     * @param $object
+     * @param array $mergedMapping
+     * @return array|boolean
+     */
     public function recursiveGetActiveGroupCollectionMapping($object, $mergedMapping = [])
     {
         if (!$object) {
-            return;
+            return false;
```

Comment:
Please do not alter the return type - previously it returned `null`, now `false`.